pkg/api/handlers/compat/images_build.go

```diff
@@ -399,20 +399,46 @@ func BuildImage(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	// Docker's newer clients popuates `cacheFrom` and `cacheTo` parameter
```

Comment:
If this will be removed when changes to Bulidah are completed, we need a big old "TODO" here and pointers to appropriate issues and PRs.

Reply:
Done.